subscribie/blueprints/admin/templates/admin/settings/rename_shop.html

```diff
@@ -23,21 +24,69 @@ <h2>Change Your Shop Web Address</h2>
           <p>The URL of your shop will be changed as long as the name is available
               and the name contains only letters and numbers</p>
 	  <p> Your current web address is: {{ SERVER_NAME }}</p>	
-          <form action="" method="POST" action="/">
+          <div>
             <div class="form-group">
      
               <label for="shop">New name:</label>
-              <input type="text" name="new_name" class="form-control" id="shop" 
+              <input type="text" name="new_name" class="form-control" id="new_name"
                      aria-describedby="shopHelp" placeholder="{{ SERVER_NAME }}"  required>
             </div>
             <button type="submit" id="submit" class="btn btn-primary btn-block mb-3">Save</button>
-          </form>
+          </div>
         </div>
 
       </div>
 
     </div> <!-- end container-->
   </div> <!-- end section-->
+
+<script>
```

Comment:
@joeltejeda :)  this is the rename site client site waiting you might want to review /look at :+1: